sed/core.py

```diff
@@ -98,15 +150,43 @@ def compute(
             axes: [description].
             nbins: [description].
             ranges: [description].
-            binDict: [description].
             pbar: [description].
             jittered: [description].
             pbenv: [description].
 
         Returns:
             [description]
         """
-        pass
+        assert (
+            self._dataframe is not None
+        ), "dataframe needs to be loaded first!"
+
+        if nCores is None:
+            nCores = self._config["num_cores"]
+
+        df = self._dataframe
+
+        if jittered:
```

Comment:
If we apply the jitter at last (see comment above) this would be where to do it I guess.

Reply:
Possibly, but as commented above, I would not modify the df in the binning function.